packages/subgraph/schema.graphql

```diff
@@ -399,27 +399,27 @@ type AdminPlugin implements IPluginInstallation @entity {
   pluginAddress: Bytes!
 
   proposals: [AdminProposal!]! @derivedFrom(field: "plugin")
-  adminstrators: [AdminstratorAdminPlugin!]! @derivedFrom(field: "plugin")
+  administrators: [AdministratorAdminPlugin!]! @derivedFrom(field: "plugin")
 }
 
-type Adminstrator @entity {
+type Administrator @entity(immutable: true) {
   id: ID! # address
   address: String # address as string to facilitate filtering by address on the UI
-  proposals: [AdminProposal!]! @derivedFrom(field: "adminstrator")
-  plugins: [AdminstratorAdminPlugin!]! @derivedFrom(field: "administrator")
+  proposals: [AdminProposal!]! @derivedFrom(field: "administrator")
+  plugins: [AdministratorAdminPlugin!]! @derivedFrom(field: "administrator")
```

Comment:
why `AdministratorAdminPlugin` ? shouldn't it be `AdminPlugin` directly ? Feels like `AdminstratorAdminPlugin` makes no sense(definately can be done without it).

Reply:
No because `AdministratorAdminPlugin` is the many to many mapping between Administrators and the AdminPlugins